Documentation/Signoffs/IdentificationSystem.md

```diff
@@ -0,0 +1,68 @@
+# Identification Subsystem  
+
+## Subsystem Function  
+
+This Subsystem's role in the team's design is to scan barcodes on school devices before being placed and when being taken from the machine. The subsystem also includes a RFID Card Reader to verify and store students' identity. The PC will then receive information from the modules and store information in a database for our customer. 
+
+## Constraints  
+
+* Barcode Scanners shall be low power and consume no greater than 5 V 0.5 A. This is due to being powered by the miniPC. Common computers only output a max rating of 5 V 0.5 A.  
+^ Source: https://resources.pcb.cadence.com/blog/2020-what-are-the-maximum-power-output-and-data-transfer-rates-for-the-usb-standards  
+
+* Barcode Scanners shall be able to decode Code-128 barcodes 
+
+* RFID Card Scanner shall be low power and consume no greater than 5 V 0.5 A   
+
+* RFID Card Scanner shall operate on 13.56 MHz commonly used for higher security access badges  
+
+* Barcode Scanners and the RFID Card reader shall be powered by a USB port to communicate with the miniPC  
+
+* A Barcode Scanner and Card Reader will be monitered by USB voltmeter to ensure correct operation  
+
+* Barcodes shall be of decent quality to follow ISO 15416 standard. Assures that the codes can be read by any scanner  
+
+* The MiniPC shall run on Windows to provide the needed applications for PLC and Database
+
+* The MiniPC shall occupy no larger than a 6x6 in. space to fit in the machine
+
+* The MiniPC will provide an ethernet port to communicate with the PLC
+
+* Shall include a USB Hub to provide enough ports for modules
+
+* Shall include a port to connect to the UI system  
+
+## Schematic  
```

Comment:
What is the miniPC? is this a specific device? (Probably not a problem and simply a misunderstanding on my part)

Reply:
How will this communicate with the PLC? What does it need to communicate? Communication is going to be the hardest part of this system.